src/dmd/identifier.h

```diff
@@ -18,16 +18,18 @@ class Identifier : public RootObject
 private:
     int value;
     DString string;
+    bool isAnonymous_;
```

Comment:
Put is between `value` and `string`. On x86_64 the size will remain at 32 Bytes as the new bool is now in a 8 Bytes hole. On i386 that doesn't change anything.

Reply:
Done.

Reply:
indeed verified, although the review was made on the header.